components/accordion/stories/template.js

```diff
@@ -50,6 +50,7 @@ export const AccordionItem = ({
         </button>
         ${Icon({
           iconName: "ChevronRight100",
+          size: `${iconSize}`,
```

Comment:
I think this can be simplified as just:
```suggestion
          size: iconSize,
```